lib/plugins/aws/package/compile/events/cloudFront.js

```diff
@@ -196,9 +196,9 @@ class AwsCompileCloudFrontEvents {
               if (!behaviorConfig) return false;
               return (
                 behaviorConfig.ForwardedValues ||
-                behaviorConfig.MinTTL !== null ||
-                behaviorConfig.MaxTTL !== null ||
-                behaviorConfig.DefaultTTL !== null
+                behaviorConfig.MinTTL !== undefined ||
```

Comment:
What do you think about instead of checking for undefined, we'll check for `!= null`, which should cover both `null` and `undefined` values properly in that check?

Reply:
`null` should not be allowed at all – CloudFormation will complain that `null` is not a valid value.